components/FieldTripListPage/index.js

```diff
@@ -44,17 +49,25 @@ const FieldTripListPage = () => {
     const day = date.getDate();
     const month = date.getMonth() + 1;
     const year = date.getFullYear();
-    return `${day}.${month}.${year}`;
+    return `${day}.${month}.${year}`; // return formatted date
   };
```

Comment:
recently i got an advice from Marc i could handle to you:

instead of assembling your own date (like i did exact the same) try this:
`const date = new Date().toLocaleDateString('de-DE', { dateStyle: "medium" })`

Reply:
thanks i will look into it